Jenkinsfile

```diff
@@ -209,6 +209,7 @@ pipeline {
                             build-server-rpm pmm-server
                             build-server-rpm pmm-update
                             build-server-rpm dbaas-controller
+                            build-server-rpm dbaas-tools
```

Comment:
This change, and the one below should be in a separate PR too – we will merge it, but we will not merge branch and submodule update

Reply:
What's the sense of it?

Reply:
We will merge that line